### ISSUE TITLE
Scam

### ISSUE THREAD

**Issue by maxxxxing** [Pymmdrza/SoloMiner#7]
The owner somehow closed my issue I think, there is a Trojan and a malware in this program. Be careful.

**Comment by Pymmdrza** [comment]
It is very good that I published the open source version of this tool so that some ignorant people can be answered automatically.
(those who have a little knowledge can find out what funny topic this annoying person has opened by checking the source)

**Comment by fled-dev** [comment]
The code appears functional. It has the components needed for solo Bitcoin mining: connecting to a mining pool, building blocks, and attempting to solve them.

**Comment by maxxxxing** [comment]
> The code appears functional. It has the components needed for solo Bitcoin mining: connecting to a mining pool, building blocks, and attempting to solve them.

Yes, but does it really transfer the bitcoin into your wallet, have you tried it yourself?
